source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -0,0 +1,318 @@
+#include "stdafx.h"
+#include "CppUnitTest.h"
+#include "MarkDownParser.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+using namespace AdaptiveCards;
+using namespace std;
+
+namespace AdaptiveCardsSharedModelUnitTest
+{
+    TEST_CLASS(EmphasisLeftDelimterTest)
+    {
+        TEST_METHOD(LefDelimiterTest)
+        {
+            MarkDownParser blck("*foo bar*");
```

Comment:
>blck [](start = 27, length = 4)

Whis is this variable called blck? Let's try to use more meaningful names for variables.